.github/workflows/ci.yaml

```diff
@@ -152,10 +152,11 @@ jobs:
     runs-on: ubuntu-latest
     name: Release dev
     needs: [dockerfile-lint, helm-tests, unit-tests, e2e-tests, build]
-    if: github.ref == 'refs/heads/main'
     steps:
       - uses: actions/checkout@v4
         with:
+          ref: ${{ github.head_ref }}
```

Comment:
Required to get the full barnch history so that semantic-release can calc the version